src/Symfony/Bundle/WebProfilerBundle/Tests/EventListener/TurboDriveCspListenerTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebProfilerBundle\Tests\EventListener;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\PsrHttpMessage\Tests\Fixtures\App\Kernel;
```

Comment:
Not sure you need this.

https://github.com/symfony/symfony/blob/7.3/src/Symfony/Bundle/WebProfilerBundle/Tests/EventListener/WebDebugToolbarListenerTest.php

Reply:
I was talking about the Symfony bridge PsrHttpMesssage Kernel here...  this is why i linked the other similar test. 

Verry sorry if you understood i suggested to remove the test entirely.